Common/UnitDefinitions/DynamicViscosity.json

```diff
@@ -79,7 +79,19 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "lbf·s/ft²" ]
+          "Abbreviations": ["lbf·s/ft²"]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundFootSecond",
```

Comment:
Should this perhaps be named PoundPerFootSecond?
Ref: https://www.aqua-calc.com/what-is/dynamic-viscosity/pound-per-foot-second